ui/admin/app/components/form/policy/index.hbs

```diff
@@ -0,0 +1,81 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<Rose::Form
+  @onSubmit={{@submit}}
+  @cancel={{@cancel}}
+  @disabled={{@model.isSaving}}
+  @showEditToggle={{if @model.isNew false true}}
+  as |form|
+>
+  <Hds::Form::TextInput::Field
+    @isOptional={{true}}
+    @value={{@model.name}}
+    @isInvalid={{@model.errors.name}}
+    @type='text'
+    name='name'
+    disabled={{form.disabled}}
+    {{on 'input' (set-from-event @model 'name')}}
+    as |F|
+  >
+    <F.Label>{{t 'form.name.label'}}</F.Label>
+    <F.HelperText>{{t 'form.name.help'}}</F.HelperText>
+    {{#if @model.errors.name}}
+      <F.Error as |E|>
+        {{#each @model.errors.name as |error|}}
+          <E.Message>{{error.message}}</E.Message>
+        {{/each}}
+      </F.Error>
+    {{/if}}
+  </Hds::Form::TextInput::Field>
+  <Hds::Form::Textarea::Field
+    @isOptional={{true}}
+    @value={{@model.description}}
+    @isInvalid={{@model.errors.description}}
+    name='description'
+    disabled={{form.disabled}}
+    as |F|
+  >
+    <F.Label>{{t 'form.description.label'}}</F.Label>
+    <F.HelperText>{{t 'form.description.help'}}</F.HelperText>
+    {{#if @model.errors.description}}
+      <F.Error as |E|>
+        {{#each @model.errors.description as |error|}}
+          <E.Message>{{error.message}}</E.Message>
+        {{/each}}
+      </F.Error>
+    {{/if}}
+  </Hds::Form::Textarea::Field>
+
+  {{! retention policy }}
+  <Form::Policy::PolicySelection
+    @name='retention_policy'
+    @disabled={{form.disabled}}
+    @model={{@model}}
+    @options={{this.listRententionOptions}}
+    @customInputName='retain_for'
+    @inputValue={{@model.retain_for.days}}
+    @selectedOption={{this.selectRetentionPolicyType}}
+  />
+
+  {{! deletion policy }}
+  <Form::Policy::PolicySelection
+    @name='deletion_policy'
+    @disabled={{form.disabled}}
```

Comment:
One thing i'm noticing from the designs that is missing is that if the retention policy is selected to be "forever" then the deletion policy field should be disabled.
<img width="629" alt="Screenshot 2024-01-11 at 4 42 32 PM" src="https://github.com/hashicorp/boundary-ui/assets/107949262/4b48693d-ffea-4e5c-87a3-49b79cc1b854">

Reply:
ah good catch, I only limited the list, now it should be disabled too